hack/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Original script found at: https://github.com/morvencao/kube-mutating-webhook-tutorial/blob/master/deployment/webhook-create-signed-cert.sh
```

Comment:
I've found this here https://raw.githubusercontent.com/openshift/multus-admission-controller/master/hack/webhook-create-signed-cert.sh

Reply:
this makes me think if this is a bug and the operator should take care or check that the secret is created with the tls certificates

Reply:
filed a bug because I could see that the kuryr code is creating the secret for its admission webhook
https://bugzilla.redhat.com/show_bug.cgi?id=1845033

Reply:
The secret is created by the service-serving-cert controller, part of the openshift control plane.

Reply:
I closed the bug, thanks for the clarifications from you and Dan Winship